### PR TITLE
installable/cleanup

### PR DESCRIPTION
**Pull request by fgregg**
I stripped a lot away here to get this repo in shape for having in be installable in our main repo. 

there's a lot of deletions. 

i've highlighted the interesting modifcations

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python
```
Comment (fgregg) [inline setup.py:1]: this file is worth looking at.

File: irsdb/filing/apps.py [hunk 2]
```diff
@@ -0,0 +1,8 @@
+from django.apps import AppConfig
```
Comment (fgregg) [inline irsdb/filing/apps.py]: add apps.py to the right repos allows them to be separately installable apps.

File: irsdb/filing/management/commands/enter_yearly_submissions.py [hunk 3]
```diff
@@ -25,7 +25,11 @@ def handle(self, *args, **options):
         for year in options['year']:
             local_file_path = os.path.join(INDEX_DIRECTORY, "index_%s.csv" % year)
 
+            if not os.path.exists(local_file_path):
```
Comment (fgregg) [inline irsdb/filing/management/commands/enter_yearly_submissions.py:27]: irsx can now install things, so we take advantage of that.

File: irsdb/metadata/management/commands/generate_schemas_from_metadata.py [hunk 4]
```diff
@@ -116,7 +117,6 @@ def write_sked(self, schedule):
 
             variables_in_this_part = Variable.objects.filter(
                 parent_sked_part=form_part.parent_sked_part,
-                version_end__in=['','2016', '2017', '2018'],
```
Comment (fgregg) [inline irsdb/metadata/management/commands/generate_schemas_from_metadata.py:119]: this is what was leading to the `TypeError: pf_part_xii() got unexpected keyword arguments: 'PctSct4940OrgNtInvstIncmAmt', 'AdjstdQlfyngDstrAmt'`

File: irsdb/metadata/management/commands/load_metadata.py [hunk 5]
```diff
@@ -1,29 +1,18 @@
...
 REPORT_COUNT = 100
 CANONICAL_VERSION = '2016v3.0'
 
+
 class Command(BaseCommand):
     help = """
                 Erase and reload the metadata tables, one at a time.
             """
 
-    def read_blacklist(self):
```
Comment (fgregg) [inline irsdb/metadata/management/commands/load_metadata.py:18]: this didn't seem to be doing anything

File: irsdb/return/management/commands/load_filings.py [hunk 6]
```diff
@@ -36,6 +26,9 @@ class Command(BaseCommand):
     def add_arguments(self, parser):
         # Positional arguments
         parser.add_argument("year", nargs=1, type=int)
+        parser.add_argument(
+            "--file", dest="file", help="Path of CSV file to import", required=False
```
Comment (fgregg) [inline irsdb/return/management/commands/load_filings.py:29]: taught this argument to take in a file

Comment (antidipyramid) [reply irsdb/return/management/commands/load_filings.py:29]: This is a great change.

**Review by antidipyramid — APPROVED** [review]

**Pull request merged**